maxim/filter_objects.py

```
@@ -68,22 +69,33 @@ def parse_incoming_query(incoming_query: str) -> List[RuleType]:
-                        value = parsed_value
-                except (json.JSONDecodeError, ValueError):
-                    pass
+                    parsed = True
+                if not parsed:
+                    try:
+                        value = int(value)
+                        parsed = True
+                    except ValueError:
+                        pass
+                if not parsed:
+                    if type(value) == bool:
```

Flake8 (7.2.0)

[error] do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`

(E721)